config/initializers/devise.rb

```diff
@@ -266,7 +266,7 @@
   # config.navigational_formats = ['*/*', :html, :turbo_stream]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = :get # get method to work with link_to
```

Comment:
I also see that you have log out link with `'data-turbo-method': :delete`. Which one is it?
For the record, I'd prefer to use delete as the method. We are destroying a session.

Reply:
The problem was that link_to uses 'get' method by default, I've tried to change it via turbo but didn't work so I ended up changing the default method for devise sign out.

I've fixed / changed the solution to this problem by changing the method request via rails (don't know why I didn't do that in the first place, sorry) check this [commit](https://github.com/AguirreCampiNicolas/onboarding/pull/3/commits/9932501760dbe1dc0430ff69d1d8d997e1c9ec1a)